 *** caught segfault ***
address (nil), cause 'unknown'
An irrecoverable exception occurred. R is aborting now ...
/home/runner/work/_temp/cdb57c13-deb2-45e7-b88f-5f0601b74483.sh: line 1:  9396 Segmentation fault      (core dumped) Rscript testthat.R
##[error]Process completed with exit code 139.
